src/backend/metal/src/internal.rs

```diff
@@ -6,26 +6,45 @@ use std::collections::HashMap;
 use std::path::Path;
 
 
+#[derive(Debug)]
+pub struct _ClearVertex {
+    pub pos: [f32; 4],
+}
+
 #[derive(Debug)]
 pub struct BlitVertex {
     pub uv: [f32; 4],
     pub pos: [f32; 4],
 }
 
 #[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
-pub enum BlitChannel {
+pub enum Channel {
     Float,
     Int,
     Uint,
 }
 
-pub type BlitKey = (metal::MTLTextureType, metal::MTLPixelFormat, BlitChannel);
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub struct ClearMask {
```

Comment:
Should these (and others) be `pub(crate)`?

Reply:
I often see this over-used and noisy. In our case, the whole module is private, so unless `lib.rs` re-exports it explicitly, it stays hidden within the crate.